src/transformers/models/trajectory_transformer/configuration_trajectory_transformer.py

```diff
@@ -0,0 +1,167 @@
+# coding=utf-8
+# Copyright 2022 Michael Janner and Qiyang Li and Sergey Levine and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TrajectoryTransformer model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+TRAJECTORY_TRANSFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "CarlCochet/trajectory-transformer-halfcheetah-medium-v2": "https://huggingface.co/CarlCochet/trajectory-transformer-halfcheetah-medium-v2/resolve/main/config.json",
```

Comment:
Before merging, it would be nice to upload these to the author's organization. Have you been in contact with the authors? It would be nice to ask them if they want to have an organization on the hub and to host these weights there.

Let me know if you'd like some pointers.

Reply:
I have indeed been in contact with the authors and could probably ask if they want to do that, however their pretrained checkpoints are not compatible with this model. I had to modify them on my end to make them compatible with transformers, so I'm not sure how that would work.